src/api/app/components/notification_action_description_component.rb

```diff
@@ -54,4 +50,10 @@ def bs_request
                       @notification.notifiable.commentable
                     end
   end
+
+  def event_decision_description
+    decision_text = @notification.notifiable.kind.sub(/ed$/, '')
```

Comment:
I find this even less clear than  the previous one, sorry... To me `OOP` to the rescue meant something else...

Reply:
I see. Is it now better?

Reply:
My point is: in order to save a case branch, we're getting more complex code in exchange. I think this change is not worth it, sorry. Those two `when` branches were easy to understand as they were before...